### PR TITLE
Feature/#1053 permission buttons

### PR DESCRIPTION
**Pull request by ciaranschutte**

### CODE REVIEW

**Comment by ciaranschutte** [comment]
Fixed

File: src/uikit/Button.js [hunk 1]
```diff
@@ -123,3 +122,12 @@ export const LargeTealActionButton = styled(TealActionButton)`
...
   border-radius: 18px;
 `;
+
+export const DisabledButton = ({ theme }) => css`
+  background-color: ${theme.backgroundGrey};
+  color: ${theme.borderGrey};
+  &:hover:disabled {
+    background-color: ${theme.backgroundGrey};
+  }
+  cursor: not-allowed;
```
[the remark this answers is not in the document]
Comment (ciaranschutte) [reply src/uikit/Button.js]: Fixing name change

TealActionButton is in use in a lot of places. Let's leave adding it directly until we have time to test it properly.

**Comment by ciaranschutte** [comment]
@hlminh2000 updated :)

**Comment by hlminh2000** [comment]
oh tiny thing:
screen shot 2019-01-18 at 4 06 07 pm

**Pull request merged**